.gitignore

```diff
@@ -106,3 +106,166 @@ ENV/
 var_holder.json
 
 data/
+
+# vs code
+.vscode/
```

Comment:
if this is ignore, can we remove it from this PR ?

Reply:
also, there is a lot of entries in this file that can be removed, but hey, we don't care, it's ok, just a generic file